client/card-readers/index.js

```diff
@@ -0,0 +1,52 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies.
+ */
+import Page from 'components/page';
+import ReadersList from './list';
+import ReadersSettings from './settings';
+import { TabPanel } from '@wordpress/components';
+
+import './style.scss';
+
+export const ConnectedReaders = () => {
+	return (
+		<Page>
+			<TabPanel
+				className="wcpay-card-readers-page"
+				activeClass="active-tab"
+				tabs={ [
+					{
+						name: 'connected-readers',
+						title: __(
+							'Connected readers',
+							'woocommerce-payments'
+						),
+						className: 'connected-readers-list',
+					},
+					{
+						name: 'receipt-details',
+						title: __( 'Receipt details', 'woocommerce-payments' ),
+						className: 'connected-readers-receipt-details',
+					},
+				] }
+			>
+				{ ( tab ) => {
```

Comment:
A wishful comment: Is there a built-in way to update the browser's address bar on switching tabs? That'd make a tab's URL shareable.

If such functionality is not already baked into WordPress or WooCommerce components, we don't need to bother too much.

Reply:
Unfortunately, the tab component has no built-in routes to be updated from the browser's address.